build.gradle

```diff
@@ -23,7 +23,7 @@ buildscript {
         }
 
         common_utils_version = System.getProperty("common_utils.version", opensearch_build)
-        kotlin_version = System.getProperty("kotlin.version", "1.8.21")
+        kotlin_version = System.getProperty("kotlin.version", "1.9.23")
```

Comment:
@Hailong-am added Kotlin to ml-commons in this PR https://github.com/opensearch-project/ml-commons/pull/1845 for search alert tool

Hailong, do we really need this dependency now?

Reply:
yes, we still need it. You can see https://github.com/opensearch-project/skills/issues/116 for more context. As long as ml-commons has dependency on common-utils, kotlin dependency is needed for skills repo.